test/python/WMCore_t/WMSpec_t/Steps_t/Executors_t/DQMUpload_t.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import print_function
+import logging
+from future import standard_library
+standard_library.install_aliases()
+import urllib.request
+
+import unittest
+
+import os
+from functools import reduce
+from hashlib import md5
+
+from WMCore.WMSpec.Steps.Executors.DQMUpload import DQMUpload
+
+class StepPatch():
+    def __init__(self, upload=None):
+        self.upload = upload
+
+class UploadPatch():
+    def __init__(self):
+        self.proxy = False
+
+class DQMUpload_t(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    @attr("integration")
```

Comment:
@mapellidario I think you missed the
```
from nose.plugins.attrib import attr
```

Reply:
aaaaaa you are correct, sorry! thanks for th suggestions!